### DISCUSSION TITLE
Upgrading to Compose V2

### DISCUSSION THREAD

**Discussion by alezanott** [teslamate-org/teslamate#3570] [Q&A] [answered]
Good evening everyone,
I'm currently running Teslamate on a RPI 4. I want to upgrade to Docker Compose V2. On the online guide I can't find a proper solution for this upgrade, since I want to maintain all the data stored for Teslamate.

Any idea?

**Comment by cwanja** [comment]
Upgrading Docker Compose should not impact your TeslaMate install. As always, backup your data.

Please follow the links in the docs.

**Reply by cwanja** [reply, marked as answer]
https://docs.docker.com/compose/migrate/

**Reply by DrMichael** [reply]
Worked well for me.
